### PR TITLE
Bumping the dev deps 2018-12-21 edition

### PR DESCRIPTION
**Pull request by etpinard**
cc @plotly/plotly_js

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1044,6 +1060,48 @@
...
+      "dependencies": {
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
+          "dev": true
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 2]
```diff
@@ -1044,6 +1060,48 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 3]
```diff
@@ -2479,6 +2547,48 @@
...
+      "dependencies": {
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
+          "dev": true
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 4]
```diff
@@ -2479,6 +2547,48 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 5]
```diff
@@ -2517,6 +2627,46 @@
...
         },
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
+          "dev": true
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 6]
```diff
@@ -2517,6 +2627,46 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 7]
```diff
@@ -5194,6 +5349,44 @@
...
+      },
+      "dependencies": {
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE="
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 8]
```diff
@@ -5194,6 +5349,44 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 9]
```diff
@@ -5741,6 +5953,48 @@
...
+      "dependencies": {
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE=",
+          "dev": true
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 10]
```diff
@@ -5741,6 +5953,48 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 11]
```diff
@@ -8902,6 +9172,44 @@
...
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 12]
```diff
@@ -8902,6 +9172,44 @@
...
+          }
+        },
+        "isarray": {
+          "version": "1.0.0",
+          "resolved": "https://registry.npmjs.org/isarray/-/isarray-1.0.0.tgz",
+          "integrity": "sha1-u5NdSFgsuhaMBoNJV6VKPgcSTxE="
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 13]
```diff
@@ -11767,6 +12077,40 @@
             "parse-glob": "^3.0.4",
             "regex-cache": "^0.4.2"
           }
+        },
+        "readable-stream": {
+          "version": "2.3.6",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

File: package-lock.json [hunk 14]
```diff
@@ -11767,6 +12077,40 @@
...
+            "isarray": "~1.0.0",
+            "process-nextick-args": "~2.0.0",
+            "safe-buffer": "~5.1.1",
+            "string_decoder": "~1.1.1",
+            "util-deprecate": "~1.0.1"
+          }
+        },
+        "string_decoder": {
+          "version": "1.1.1",
+          "resolved": "http://registry.npmjs.org/string_decoder/-/string_decoder-1.1.1.tgz",
```
Comment (archmoj) [inline package-lock.json]: http > https

**Comment by etpinard** [comment]
Good eye @archmoj -> https://github.com/plotly/plotly.js/pull/3367/commits/31066fa06daaef1269f23c0a69ab401d2f74e3c7

File: package.json [hunk 15]
```diff
@@ -147,15 +147,15 @@
     "madge": "^3.3.0",
     "minify-stream": "^1.2.0",
     "minimist": "^1.2.0",
-    "node-sass": "^4.10.0",
-    "npm-link-check": "^2.0.0",
+    "node-sass": "^4.11.0",
+    "npm-link-check": "^3.0.0",
```
Comment (archmoj) [inline package.json]: Cool module by @etpinard

**Comment by archmoj** [comment]
Nicely done! :dancer: 
It would be great if we could follow up the `cwise` PR #3267 for the next version.

**Pull request merged**